specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Describes the result of the request to list Microsoft.Management operations.",
+      "properties": {
+        "value": {
+          "description": "List of operations supported by the Microsoft.Management resource provider.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CheckNameAvailabilityResult": {
+      "description": "Describes the result of the request to check management group name availability.",
+      "properties": {
+        "nameAvailable": {
+          "description": "Required. True indicates name is valid and available. False indicates the name is invalid, unavailable, or both.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "reason": {
+          "description": "Required if nameAvailable == false. Invalid indicates the name provided does not match the resource provider's naming requirements (incorrect length, unsupported characters, etc.) AlreadyExists indicates that the name is already in use and is therefore unavailable.",
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "AlreadyExists"
+          ],
+          "x-ms-enum": {
+            "name": "Reason",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Invalid"
+              },
+              {
+                "value": "AlreadyExists"
+              }
+            ]
+          },
+          "readOnly": true
+        },
+        "message": {
+          "description": "Required if nameAvailable == false. Localized. If reason == invalid, provide the user with the reason why the given name is invalid, and provide the resource naming requirements so that the user can select a valid name. If reason == AlreadyExists, explain that is already in use, and direct them to select a different name.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "TenantBackfillStatusResult": {
+      "description": "The tenant backfill status",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "status": {
+          "description": "The status of the Tenant Backfill",
+          "type": "string",
+          "enum": [
+            "NotStarted",
+            "NotStartedButGroupsExist",
+            "Started",
+            "Failed",
+            "Cancelled",
```

Comment:
>Cancelled [](http://example.com/codeflow?start=13&length=9)

For consistency with ARM provisioning states, recommend using Canceled and Succeeded instead of Cancelled and Completed.